runtime/cudaq/platform/pasqal/PasqalServerHelper.cpp

```diff
@@ -52,50 +55,60 @@ RestHeaders PasqalServerHelper::getHeaders() {
 
 ServerJobPayload
 PasqalServerHelper::createJob(std::vector<KernelExecution> &circuitCodes) {
-  ServerJobPayload ret;
-  std::vector<ServerMessage> &tasks = std::get<2>(ret);
+  std::vector<ServerMessage> tasks;
+
   for (auto &circuitCode : circuitCodes) {
-    // Construct the job message
-    ServerMessage taskRequest;
+    ServerMessage message;
+    message["name"] = circuitCode.name;
+    message["device"] = backendConfig.at("machine");
```

Comment:
I actually kept `machine` in the cloud side API, as we typically use `device` for the full spec. We could alternatively use `device_type` here.

Reply:
I think `device_type` may create more confusion while `machine` makes more sense with Fresnel and others. So I will keep it as `machine`